src/mainviews/list-jobs.tsx

```diff
@@ -145,8 +146,13 @@ export function ListJobsTable(props: IListJobsTableProps): JSX.Element {
     !deletedRows.has(job.job_id);
 
   const emptyRowMessage = useMemo(
-    () => trans.__('There are no notebook jobs.'),
-    [trans]
+    () =>
+      props.emptyRowMessage ??
+      trans.__(
+        'There are no notebook jobs. Notebook jobs run notebooks and other files in the background, either immediately or on a schedule. ' +
```

Comment:
```suggestion
        'There are no notebook jobs. Notebook jobs run files in the background, immediately or on a schedule. ' +
```

Copy edits for brevity